components/infobox/wikis/mobilelegends/infobox_item_custom.lua

```diff
@@ -0,0 +1,344 @@
+		}
+		for category, requiredArg in pairs(possibleCategories) do
+			if String.isNotEmpty(_args[requiredArg]) then
+				table.insert(_categories, category)
+			end
+		end
+
+		return _categories
+	end
+end
+
+function CustomItem.nameDisplay()
+	return _args.itemname
+end
+
+function CustomItem._getCostDisplay()
+	local costs = Item:getAllArgsForBase(_args, 'itemcost')
+
+	local innerDiv = mw.html.create('div')
+		:css('display', 'inline-block')
+		:css('padding', '0px 3px')
+		:css('border-radius', '4px')
+		:addClass('placement-darkgrey')
+		:wikitext(table.concat(costs, '&nbsp;/&nbsp;'))
+	local outerDiv = mw.html.create('div')
+		:wikitext(Template.safeExpand(
+				_frame,
+				'icons',
+				{'gold', size = '21px'},
+				''
+			) .. ' ' .. tostring(innerDiv)
+		)
+	local display = tostring(outerDiv)
+
+	if String.isNotEmpty(_args.recipecost) then
+		innerDiv = mw.html.create('div')
+			:css('display', 'inline-block')
+			:css('padding', '0px 3px')
+			:css('border-radius', '4px')
+			:addClass('placement-darkgrey')
+			:wikitext('(' .. _args.recipecost .. ')')
+		outerDiv = mw.html.create('div')
+			:css('padding-top', '3px')
+			:wikitext(Template.safeExpand(
+					_frame,
+					'icons',
+					{'recipe', size = '21px'},
+					''
+				) .. ' ' .. tostring(innerDiv)
+			)
+		display = display .. tostring(outerDiv)
+	end
+
+	return {display}
+end
+
+function CustomItem._attributeIcons(attributeType)
+	_args[attributeType] = _args[attributeType] or 0
+	local attributes = Item:getAllArgsForBase(_args, attributeType)
+	return Template.safeExpand(_frame, 'AttributeIcon', {attributeType}, '')
+		.. '<br><b>+ ' .. table.concat(attributes, '/ ') .. '</b>'
+end
+
+function CustomItem._positiveConcatedArgsForBase(base)
+	if String.isNotEmpty(_args[base]) then
+		local foundArgs = Item:getAllArgsForBase(_args, base)
+		return '+ ' .. table.concat(foundArgs, '&nbsp;/&nbsp;')
+	end
+end
+
+function CustomItem._negativeConcatedArgsForBase(base)
+	if String.isNotEmpty(_args[base]) then
+		local foundArgs = Item:getAllArgsForBase(_args, base)
+		return '- ' .. table.concat(foundArgs, '&nbsp;/&nbsp;')
+	end
+end
+
+function CustomItem._positivePercentDisplay(base)
+	if String.isNotEmpty(_args[base]) then
+		local number = tonumber(_args[base])
+		number = number * 100
+		return '+ ' .. number .. '%'
+	end
+end
+
+function CustomItem._manaLossDisplay()
+	local display = ''
+	if String.isNotEmpty(_args['mana loss']) then
+		display = (tonumber(_args['mana loss']) + 100) .. '%'
+	end
+	if String.isNotEmpty(_args.manaloss) then
+		display = display .. (tonumber(_args.manaloss) + 100) .. '%'
+	end
+	if display ~= '' then
+		return '+ ' .. display
+	end
+end
+
+function CustomItem._movementSpeedDisplay()
+	local display = ''
+	if String.isNotEmpty(_args.movespeed) then
+		display = _args.movespeed
+	end
+	if String.isNotEmpty(_args.movespeedmult) then
+		display = display .. (tonumber(_args.movespeedmult) + 100) .. '%'
+	end
```

Comment:
same as above as well (movespeedmult = Movement Speed Multiplier)

Reply:
If you give use the parameters `|movespeed=3` and `|movespeedmult=50` it will display as `350%`, which doesn't seem correct to me. 
Also a none-numeric movespeedmult will throw lua error, better to catch it or ignore it.